src/pages/About.js

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable react/style-prop-object */
+/* eslint-disable jsx-a11y/iframe-has-title */
+/* eslint-disable max-lines-per-function */
```

Comment:
Is there a reason you need all of these error disabling comments?
If the iframe needs a 'title' attribute then just give it one